compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -160,6 +170,27 @@ internal class ComposeContainer(
         layers.fastForEach(DesktopComposeSceneLayer::onChangeWindowFocus)
     }
 
+    override fun windowOpened(e: WindowEvent) {
+        lifecycleRegistry.handleLifecycleEvent(Lifecycle.Event.ON_RESUME)
+    }
+
+    override fun windowClosing(e: WindowEvent) = Unit
+
+    override fun windowClosed(e: WindowEvent) {
+        lifecycleRegistry.handleLifecycleEvent(Lifecycle.Event.ON_STOP)
+    }
+
+    override fun windowIconified(e: WindowEvent) {
+        lifecycleRegistry.handleLifecycleEvent(Lifecycle.Event.ON_PAUSE)
```

Comment:
It seems that it fully hide Compose, so may be it better aligns with START/STOP.

PAUSE/RESUME is about being foreground. For example it should happen when some dialog window is shown above